problem_builder/tests/unit/test_instructor_tool.py

```diff
@@ -224,16 +51,15 @@ def test_student_view_template_args(self):
             'Rating Question': 'RatingBlock',
             'Long Answer': 'AnswerBlock',
         }
-        flat_block_tree = ['block{}'.format(i) for i in range(10)]
-        self.block._build_course_tree = Mock(return_value=flat_block_tree)
 
         with patch('problem_builder.instructor_tool.loader') as patched_loader:
             patched_loader.render_template.return_value = u''
             self.block.student_view()
-            patched_loader.render_template.assert_called_once_with(
-                'templates/html/instructor_tool.html',
-                {'block_choices': block_choices, 'block_tree': flat_block_tree}
-            )
+            patched_loader.render_template.assert_called_once_with('templates/html/instructor_tool.html', {
+                'block_choices': block_choices,
+                'course_blocks_api': '/api/courses/v1/blocks/',
```

Comment:
@itsjeyd I've been smacked on the wrist for doing that before for some reason..  But I totally agree.  Fixed [here](https://github.com/open-craft/problem-builder/pull/129/commits/9792c93612bde698a8d77d0f26a3c682bb4870eb#diff-abe13c221a1dbf269ccd472d143ae52aR60).

And since I squashed the commits, I also added a comment to the [`COURSE_BLOCKS_API ` declaration](https://github.com/open-craft/problem-builder/pull/129/commits/9792c93612bde698a8d77d0f26a3c682bb4870eb#diff-2e30f6544a2c3958f3ca37f29793308eR37) to explain why the trailing slash is important.  Seemed excessive to go into detail about the content switch proxy, but at least made a note.